Python/Product/Cookiecutter/CookiecutterToolWindow.cs

```diff
@@ -209,21 +209,19 @@ sealed class CookiecutterToolWindow : ToolWindowPane, IVsInfoBarUIEvents {
             _uiShell.UpdateCommandUI(0);
         }
 
-        private void OpenGeneratedFolder(string folderPath) {
-#if DEV15_OR_LATER
-            OpenInSolutionExplorer(folderPath);
-#else
-            OpenInWindowsExplorer(folderPath);
+        private void ExecuteCommand(string name, string args) {
+            try {
+                _dte.ExecuteCommand(name, args ?? string.Empty);
+            } catch (Exception ex) when (!ex.IsCriticalException()) {
+#if !DEV15_OR_LATER
+                if (name == "File.OpenFolder") {
```

Comment:
Ah, good point, I misread the ifdef. It's fine there.

The extension is totally unsupported, so I wouldn't worry about it.